### PR TITLE
Fixes syndrone health

### LINKED ISSUE #14923: Sydrone Health issues
So syndrones supposedly start with 30 health, and a maxHealth of 120, with a comment explaining that consuming other drones will make them stronger.

They have 120 health when they spawn though

### PR DESCRIPTION
**Pull request by praisenarsie**
i literally have only seen the syndrone dispenser spawn on roundend by an admin once.
also does some initalize fixes
of course there's the deeper problem on why syndrone's hp was 120 everytime so that needs to be looked into

Fixes #14923

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm [hunk 1]
```diff
@@ -16,7 +16,6 @@
 	icon_living = "drone_synd"
 	picked = TRUE //the appearence of syndrones is static, you don't get to change it.
 	health = 30
-	maxHealth = 120 //If you murder other drones and cannibalize them you can get much stronger
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: this should remain, but you should set `bruteloss = 90` below it

Comment (RandomMarine) [reply code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: That wouldn't stop the syndrone from just screwdrivering itself up to full health after it spawns.

The real solution would be to have it start with 30 maxHealth and raise that when it cannibalizes drones.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: 999 health for master eaters

**Comment by MoreRobustThanYou** [comment]
turn the floors into Syndrones!

**Comment by Cyberboss** [comment]
travis

**Comment by KorPhaeron** [comment]
This doesnt fix the issur anymore because they can immedatiely screwdriver all that damage away, your first commit had it right

**Comment by praisenarsie** [comment]
i'll do this later because brain no good

**Pull request closed without merging**